yarn.lock

```diff
@@ -4928,6 +4928,17 @@ __metadata:
   languageName: node
   linkType: hard
 
+"libram@npm:latest":
+  version: 0.8.14
+  resolution: "libram@npm:0.8.14"
+  dependencies:
+    html-entities: ^2.4.0
+  peerDependencies:
+    kolmafia: ^5.26781.0
+  checksum: 342672a9a5f2b798c3493006b79c3cd9224a7cb4425960e218bc09244b7a48b4656b6535093cdbd37e93df88bbc4d6af3edb5395bc3a5fe624ef19714f50810a
+  languageName: node
+  linkType: hard
+
```

Comment:
The libram package directly above this should be updated and this one should be removed.

Reply:
Also fixed!

Reply:
It looks like the yarn.lock file is no longer modified. While it's good at there aren't duplicate packages, you probably want 0.8.14 to be the version specified not 0.8.13.

Reply:
What's the best way to correct this without yarn upgrade kolmafia@latest overriding to latest again?

Reply:
```
yarn add libram@0.8.14
yarn dedupe libram
```
Should be sufficient here. If it also needs to be upgraded in garbo-lib or another workspace then you can do:
```
yarn workspace garbo-lib add --peer libram@0.8.14
```